DataScience/ExperimentationAzure.py

```diff
@@ -137,7 +137,7 @@ def check_system():
         unknown.append('--model')
         unknown.append(model_fp)
         unknown.append('--min_num_features')
-        unknown.append('3')
+        unknown.append('1')
```

Comment:
why this change?

Reply:
I added the min_num_features initially to ensure I don't get back zero buckets. Tyler's point was that setting it to 3 will mean we cannot have a bucket of just 1 - which can happen (depending on the data). So I changed it to 1.

Reply:
Can we rename the variable `unknown` to have a less unknown meaning...

Reply:
Changed to other_args